addons/mail/controllers/mail.py

```diff
@@ -113,6 +113,9 @@ def _redirect_to_record(cls, model, res_id, access_token=None, **kwargs):
             'active_id': res_id,
             'action': record_action.get('id'),
         }
+        menu_id = request.env['ir.ui.menu']._get_best_menu_root_for_model(model)
```

Comment:
Can you add a title "Purpose" and "Technical" to separate the 2 sections in your commit message ?

Reply:
Yes indeed the goal helps to understand the code. (updated but not pushed yet)